springfox-spring-web/build.gradle

```diff
@@ -13,6 +13,8 @@ dependencies {
   compile "io.github.classgraph:classgraph:$classGraph"
   compile libs.core
   compile libs.spring
+  compile libs.swagger2Core
```

Comment:
This is not the correct fix. There is a reason that swagger is not part of this spring-web library. We don't want to mix that concern in this library. For e.g. what about open api annotations. In that case this will become a kitchen sink of stuff and loses the efficacy of having plugins.